data/incipias/incipias first contact.txt

```diff
@@ -215,9 +215,9 @@ mission "Incipias: Help The Stranded 1"
 			choice
 				`	"I could help them."`
 					goto initiative
+				`	"How can you make such a cold-hearted decision?"`
 				`	"I understand. I'll leave them alone."`
 					goto declined
-				`	"How can you make such a cold-hearted decision?"`
 			`	"It is perhaps a laborious task for an adolescent species such as yourself to comprehend our ways, but we must place the continuation of species above the existence of individuals. If we reveal ourselves to them at this inopportune moment, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
```

Comment:
Ditto.

Reply:
I already do this below. The action is based on the player directly asking why they don't want them to give their tech. The option you're placing it above here is a separate conversation; if you add it here, it would be taking away the choice for no reason.